pages/static/src/js/groups.js

```diff
@@ -83,11 +115,18 @@ module.exports = function() {
   function addGroup(group) {
     if (group.geometry!=null) {
       var coords = group.geometry.coordinates;
-      var marker = L.marker(coords.reverse());
-
+      var marker;
+      
+      if (group.properties.group_type == 'state-organizing-committee' || group.properties.group_type == 'state-chapter') {
+        // if SOC, color red and put on top layer
+        marker = L.marker(coords.reverse(), {icon: redIcon, zIndexOffset: 1000})
+      } else {
+        marker = L.marker(coords.reverse(), {icon: blueIcon});
+      }
+      
       marker.properties = group.properties;
 
-      marker.bindPopup('<h4>'+ group.properties.name + '</h1>');
+      marker.bindPopup('<h4>'+ group.properties.name + '</h1><a href="/groups/'+ group.properties.slug +'" class="component__cta btn btn-block btn-primary uppercase ls2">Get Involved</a>');
```

Comment:
We should ideally put `/groups/` into settings config if we haven't already. Also suggest to put trailing slash in url.